modules/adkernelBidAdapter.js

```diff
@@ -97,7 +97,8 @@ export const spec = {
     {code: 'felixads'},
     {code: 'motionspots'},
     {code: 'sonic_twist'},
-    {code: 'displayioads'}
+    {code: 'displayioads'},
+    {code: 'rtbdemand_com'}
```

Comment:
andbeyond team says they are not an alias of you; please remove if in error

please see @andbeyondmedia 

#9005

Reply:
Ok, we'll contact andbeyond team for confirmation

Reply:
@patmmccann andbeyond alias has been removed